src/main/java/uk/gov/hmcts/darts/arm/util/files/BatchInputUploadFileFilenameProcessor.java

```diff
@@ -49,4 +49,8 @@ private void processFilename() {
             throw new IllegalArgumentException("Invalid batch input upload filename " + batchMetadataFilenameAndPath);
         }
     }
-}
+
+    public String getInputUploadFilename() {
```

Comment:
Unnecessary method as there is the lombok @Getter on class.

Reply:
removed